ioctl/cmd/action/action.go

```diff
@@ -266,7 +266,10 @@ func SendAction(elp action.Envelope, signer string) error {
 	if err != nil {
 		return output.NewError(0, "failed to get chain meta", err)
 	}
-	elp.SetChainID(chainMeta.GetChainID())
+	var isMiday bool
```

Comment:
TODO: not done yet
need to figure out Midway height in ioctl, and set chainID after that